### PR TITLE
Make the title metadata the H1 in markdown files

### PR DESCRIPTION
**Pull request by Ancairon**
This is a task needed prior to removing the metadata from the files

### CODE REVIEW

**Comment by ilyam8** [comment]
@Ancairon I see there are 3 different patterns (collectors readmes):

 - X with Netdata
 - X monitoring with Netdata
 - X monitoring

Is it ok?

**Comment by Ancairon** [comment]
@ilyam8 this process was automated, so I took whatever goes into Learn and made it H1, this will surface potential issues with titles as you correctly pointed out.

I assumed that the title for Learn was correct (which seems is not the case), feel free to change it if there are wrong things in there, and @cakrit might have a say on the patterns you talk about.

**Comment by ilyam8** [comment]
>  so I took whatever goes into Learn

Where did you take it from? I wonder how Squid became a web server?

File: modules/nginx/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ learn_topic_type: "References"
 learn_rel_path: "Integrations/Monitor/Webapps"
 -->
 
-# NGINX monitoring with Netdata
+# NGINX monitoring
```
Comment (cakrit) [inline modules/nginx/README.md:11]:
```suggestion
# NGINX monitoring with Netdata
```

File: modules/nginxplus/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ learn_topic_type: "References"
 learn_rel_path: "Integrations/Monitor/Webapps"
 -->
 
-# NGINX Plus monitoring with Netdata
+# NGINX Plus monitoring
```
Comment (cakrit) [inline modules/nginxplus/README.md:11]:
```suggestion
# NGINX Plus monitoring with Netdata
```

File: modules/geth/README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ learn_topic_type: "References"
 learn_rel_path: "Integrations/Monitor/Blockchain"
 -->
 
-# Geth Monitoring with Netdata
+# Go-ethereum monitoring with Netdata
```
Comment (cakrit) [inline modules/geth/README.md:11]:
```suggestion
# Geth monitoring with Netdata
```

File: modules/nginxvts/README.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ learn_topic_type: "References"
 learn_rel_path: "Integrations/Monitor/Webapps"
 -->
 
-# NGINX VTS monitoring with Netdata
+# NGINX VTS monitoring
```
Comment (cakrit) [inline modules/nginxvts/README.md:10]:
```suggestion
# NGINX VTS monitoring with Netdata
```

File: modules/weblog/README.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ learn_topic_type: "References"
 learn_rel_path: "Integrations/Monitor/Quantify logs to metrics"
 -->
 
-# Web log (Apache, NGINX, IIS) monitoring with Netdata
+# Web server log (Apache, NGINX) monitoring with Netdata
```
Comment (cakrit) [inline modules/weblog/README.md:11]:
```suggestion
# Web server log (Apache, NGINX, IIS) monitoring with Netdata
```

File: modules/windows/README.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ learn_topic_type: "References"
 learn_rel_path: "Integrations/Monitor/System metrics"
 -->
 
-# Windows machine monitoring with Netdata
+# Windows with Netdata
```
Comment (cakrit) [inline modules/windows/README.md:11]:
```suggestion
# Windows machine monitoring with Netdata
```

**Review by cakrit — APPROVED** [review]

**Comment by cakrit** [comment]
Fixed the headings, review and merge @ilyam8

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**